### PR TITLE
feat: Add Authentik SSO support

### PR DESCRIPTION
**Pull request by samip5**
I have tested that the URL's are right.

### CODE REVIEW

**Comment by samip5** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by samip5** [comment]
recheck

**Comment by rubenfiszel** [comment]
@samip5 thanks for the PR, you can take a look at OktaIcon.svelte to see how to add the Authentik svg icon

**Comment by samip5** [comment]
> @samip5 thanks for the PR, you can take a look at OktaIcon.svelte to see how to add the Authentik svg icon

My main problem is the styling needed in the start of the Svelte icon file, before anything svg. :)

File: frontend/src/lib/components/InstanceSettings.svelte [hunk 1]
```diff
@@ -213,8 +214,9 @@
 								<OAuthSetting name="gitlab" bind:value={oauths['gitlab']} />
 								<OAuthSetting name="jumpcloud" bind:value={oauths['jumpcloud']} />
 								<KeycloakSetting bind:value={oauths['keycloak']} />
+								<AuthentikSetting bind:value={oauths['authentik']} />
 								{#each Object.keys(oauths) as k}
-									{#if !['google', 'microsoft', 'github', 'gitlab', 'jumpcloud', 'okta', 'keycloak', 'slack'].includes(k) && 'login_config' in oauths[k]}
+									{#if !['authentik','google', 'microsoft', 'github', 'gitlab', 'jumpcloud', 'okta', 'keycloak', 'slack'].includes(k) && 'login_config' in oauths[k]}
```
Comment (rubenfiszel) [inline frontend/src/lib/components/InstanceSettings.svelte:219]: missing a similar check below

Comment (rubenfiszel) [reply frontend/src/lib/components/InstanceSettings.svelte:219]: line 284

Comment (samip5) [reply frontend/src/lib/components/InstanceSettings.svelte:219]: I think you meant line 284 and as such, it's done. :)

**Pull request merged**